### PR TITLE
chore: add submodule with relative URL to .gitmodules

### PR DESCRIPTION
**Pull request by evanrittenhouse**

### CODE REVIEW

**Comment by evanrittenhouse** [comment]
CC @neersighted. I think this is what we're looking for, have never really worked with submodules. From what I understand of the integration tests, I won't be able to move forward until this is merged.

**Comment by neersighted** [comment]
This is incorrect -- let me try and fix it up for you.

File: .gitmodules [hunk 1]
```diff
@@ -4,3 +4,6 @@
 [submodule "submodules/non-existent-path"]
     path = submodules/non-existent-path
     url = git@github.com:pypa/sampleproject.git
+[submodule "relative-url-submodule"]
+	path = relative-url-submodule
+	url = ./submodules/relative-url-submodule.git
```
Comment (neersighted) [inline .gitmodules]:
```suggestion
[submodule "submodules/relative-url-submodule"]
	path = submodules/relative-url-submodule
	url = ../test-fixture-vcs-repository.git
```

**Review by neersighted — CHANGES_REQUESTED** [review]

File: .gitmodules (hunk 1 above)
Comment (neersighted) [reply .gitmodules]: You'll need to make sure that the submodule ref is pinned to the branch I created for you as well.

**Comment by evanrittenhouse** [comment]
@neersighted appreciate the feedback. Updates made.

**Comment by evanrittenhouse** [comment]
@neersighted just bumping this

**Pull request closed without merging**